### PR TITLE
LPS-79301 Remove invalid build files

### PR DESCRIPTION
**Pull request by christopherbryanboyd**
Hi @gamerson , this addresses https://issues.liferay.com/browse/LPS-79301

### CODE REVIEW

**Comment by christopherbryanboyd** [comment]
ci:test

File: modules/sdk/project-templates/project-templates-activator/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sourceCompatibility = "1.8"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-activator/build.gradle:1]: Don't need this at this moment.

File: modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+import groovy.transform.Field
+import java.nio.file.Paths
+import java.nio.file.Files
+
+def isMaven = false;
+def isGradle = false;
+
+for (def o : request.getProperties()) {
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy:22]: `for (def property : request.getProperties()) {`

File: modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		if (value == 'maven') {
+			isMaven = true
+		}
+		else if (value == 'gradle') {
+			isGradle = true
+		}
+		break
+	}
+}
+def moduleDir = Paths.get(request.getOutputDirectory(), request.getArtifactId())
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy:36]: `def projectDir`

**Pull request closed without merging**